### DISCUSSION TITLE
m3u8 problem

### DISCUSSION THREAD

**Discussion by impulse888** [smplayer-dev/smplayer#1045] [Q&A]
i can get a  direct m3u8 [https://www.terabox.com/sharing/link?surl=7Nxh1E0SE30hJxQnZ7R7xw] url from my browser  with video download helper also with the stream detector and even idm.[https://www.terabox.com/share/streaming?uk=4401648140102&shareid=31630616244&type=M3U8_FLV_264_480&fid=1082543767274382&sign=7c07186a794455b6078bcbf788e4e7daafa0ac3f&timestamp=1725553376&jsToken=448654620996E4F5B6D259EDF170F82BDE8FE64DC4D1E9288C8252B8107D4AFFF5797530FE2D45605481949D080A460D96F6059EA87BE448D47965669AEF6A9E&esl=1&isplayer=1&ehps=1&clienttype=0&app_id=250528&web=1&channel=dubox] The link can be used to download an mp4 video with yt-dl.
The problem is the link will not play directly with smplayer using mpv. I have to pass the link to my browser and save the stream as m3u8.
Is there a command i should use to play without downloading? I can play other direct m3u8 urls from other sites without downloading.

**Comment by ltankush** [comment]
Brother It's not working.
